gradle/publishing.gradle

```diff
@@ -18,7 +18,6 @@ publishing {
             artifact sourcesJar
             artifact testsJar
             artifact docsJar
-            artifact shadowJar
```

Comment:
@martinda why are we removing the shadowJar?

Reply:
@cdancy Remove shadowJar artifact as it is automatically added since version [version 6.1.0](https://imperceptiblethoughts.com/shadow/changes/#v6-1-0-2020-10-05), by "shadow" [PR #357](https://github.com/johnrengelman/shadow/pull/597/files#diff-e46a28da43cd1ff7e1e02633d554e839806aa3dd42a63c4975d4291cad6caed8R24).